lib/reader.js

```diff
@@ -97,15 +97,8 @@ function Reader (props, currentStat) {
 
   self._path = self.path = path.resolve(props.path)
   if (process.platform === 'win32') {
```

Comment:
If UNC path support was added to Node core after this code was put together, then it's probably OK to nuke this whole block, except maybe a call to `path.normalize()`. @isaacs, do you have any insight to offer here?